ZenPacks/zenoss/Import4/service_definition/imp4opentsdb.json

```diff
@@ -93,6 +94,18 @@
          {
            "Name": "HBase Regionservers up",
            "Script": "{{with $rss := (child (child (parent .) \"HBase\") \"RegionServer\").Instances }}wget -q -O- http://localhost:61000/status/cluster | grep '{{$rss}} live servers'{{end}}"
+         },
+         {
+           "Name": "OpenTSDB up",
+           "Script": "/opt/opentsdb/build/tsdb uid --config=/opt/zenoss/etc/opentsdb/opentsdb.conf grep metrics jvm.uptime"
+         },
+         {
+           "Name": "DMD UUID obtained",
+           "Script": "[ -f /import4/dmd_uuid.txt ]"
```

Comment:
That sounds correct but means that our timeline is incorrect, and you actually can't start doing perf data until a model restore is done.  Can you have @rDr4g0n update the chart to reflect this?